CMakeLists.txt

```diff
@@ -10,10 +10,6 @@
 cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
 project(valhalla LANGUAGES CXX C)
 
-if("${PROJECT_SOURCE_DIR}" STREQUAL "${PROJECT_BINARY_DIR}")
```

Comment:
Do you know why this was here? This seems like an odd case to explicitly check?

Reply:
I added recently this check in 20185625a9087e5dff5a4380b6c76e11772ee473 but it does not work as expected. The best would be to use `set(CMAKE_DISABLE_SOURCE_CHANGES ON)` but it is not possible with the current tests setup.